cli.js

```diff
@@ -63,13 +69,13 @@ const requirePlugins = plugins => plugins.map(([plugin, options]) => {
 	}
 });
 
-const normalizePluginOptions = plugin => {
-	return pairs(arrify(plugin).reduce((m, v) => {
-		return typeof v === 'object' ?
-			{...m, ...v} :
-			{[v]: {}, ...m};
-	}, {}));
-};
+const normalizePluginOptions = plugin =>
+	// eslint-disable-next-line unicorn/no-array-reduce
```

Comment:
Would be great if you could fix the lint instead of just ignoring it.

Reply:
I can replace it with any other type of loop, but imo `reduce` is exactly what we're doing here — converting an array into a map.